client/src/components/ElementEditor/Header.js

```diff
@@ -185,6 +187,16 @@ class Header extends Component {
     );
   }
 
+  renderEditTabs() {
+    const { editTabs } = this.props;
+
+    if (!editTabs || !editTabs.length) {
+      return null;
+    }
+
+    return editTabs.map((tab) => <button key={tab} className="dropdown-item">{tab}</button>);
```

Comment:
Do we want to add extra class options to this now, incase eg. someone wants to put icons into it in their site?

Reply:
Cancel, that'd mean passing an array of objects instead of strings. Ignore